manifests/init.pp

```diff
@@ -83,6 +83,9 @@
 # @param version
 #   Version of Candlepin to install
 #
+# @param java_package
```

Comment:
Is there some other java package that a user would set this value to? Or that we would want to allow a user to configure this to be?

Reply:
In theory they could set it to some other package, like Oracle JVM. It also makes it easy if we update to Java 12 to do so. I strongly insist this is a parameter. Given it's not exposed directly in the installer, there isn't any downside to it.

Reply:
If we update to Java 12, its the same code change whether its a user input parameter or a hard coded value. I recall @ekohl you wanted to work toward exposing Candlepin and Pulp at the top level within the installer.

I'm not going to block on this change, I just do not get the value of exposing these sort of parameters.

Reply:
I was working on exposing https://github.com/theforeman/puppet-katello/blob/master/manifests/candlepin.pp.

Reply:
I think it's harmless although will likely go unused.